docs/AzureMarketplaceGuide.md

```diff
@@ -33,16 +33,15 @@ Note that **Company** is not required, but please enter it.
    You need several container images to run Scalar products on Kubernetes, but Azure Marketplace copies only one container image at a time. So, you need to subscribe to several software plans (repeat subscribe operation) as needed.
    * Container images that you need are the following.
         * ScalarDB
-            * ScalarDB Server Default (2vCPU, 4GiB Memory)
-            * ScalarDB Server Envoy Proxy
-            * ScalarDB GraphQL Server (optional)
-            * ScalarDB SQL Server (optional)
+            * [Deprecated] ScalarDB Server Default (2vCPU, 4GiB Memory)
+            * [Deprecated] ScalarDB GraphQL Server (optional)
+            * [Deprecated] ScalarDB SQL Server (optional)
+            * [Recommended / coming soon] ScalarDB Cluster
```

Comment:
At this time, we don't provide ScalarDB Cluster on Azure Marketplace.
So, I marked it as `coming soon`.
We will provide ScalarDB Cluster (BYOL) on the Azure Marketplace after we release v4.